gerrit_python_tools/gerrit.py

```diff
@@ -624,13 +682,30 @@ def send_upstream(self, downstream, upstream):
         approvals = self.get_approvals(downstream.SSH())
 
         # Check to see if comment has necessary approvals.
-        if not self.is_upstream_approved(approvals):
-            msg = ("Could not send to upstream: One or more labels"
-                   " not approved.")
+        if self.is_forced():
+            if not self.is_release_approved(approvals):
+                msg = "Could not send to upstream: Release not approved."
+                logger.debug("Change %s: %s" % (self.change_id, msg))
+                ssh.exec_once('gerrit review -m %s %s'
+                              % (pipes.quote(msg), self.revision))
+                return
+            msg = ("***WARNING***\n%s (%s) has requested a forced upstream "
+                   "push. Bypassing all votes except for Release...\n\n"
```

Comment:
I don't understand how simply removing the ***WARNING*** part is any better.  If anything it's worse because it makes the comment no longer stand out when you're scrolling through a potentially very long patch